### ISSUE TITLE
Cannot open include file WebView2.h with package 1.0.818.41

### ISSUE THREAD

**Issue by mariusbancila** [MicrosoftEdge/WebView2Feedback#1209]
**Description**
I have updated the NuGet package to the latest released yesterday, 1.0.818.41 and now I get the following error:
```
fatal error C1083: Cannot open include file: 'WebView2.h': No such file or directory
```
When I look at the `Common.targets` file, I see the following difference:

1.0.824-prerelease
```
<AdditionalIncludeDirectories>$(MSBuildThisFileDirectory)native\include\;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```
1.0.818.41
```
<AdditionalIncludeDirectories>$(MSBuildThisFileDirectory)include\;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```
Diff: instead of `native\include\` there is only `include\`.

There are other differences like the following:
1.0.824-prerelease
```
  <ItemGroup Condition="'$(WebView2LoaderPreference)' == 'Dynamic' And '$(WebView2ProjectKind)' == 'native'">
    <Content Include="$(MSBuildThisFileDirectory)native\$(EffectivePlatform)\WebView2Loader.dll">
      <Link>%(Filename)%(Extension)</Link>
      <PublishState>Included</PublishState>
      <Visible>False</Visible>
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </Content>
  </ItemGroup>
```
1.0.818.41
```
  <ItemGroup Condition="'$(WebView2LoaderPreference)' == 'Dynamic' And '$(WebView2ProjectKind)' == 'native'">
    <Content Include="$(MSBuildThisFileDirectory)$(EffectivePlatform)\WebView2Loader.dll">
      <Link>%(Filename)%(Extension)</Link>
      <PublishState>Included</PublishState>
      <Visible>False</Visible>
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </Content>
  </ItemGroup>
```
Diff: instead of `$(MSBuildThisFileDirectory)native\$(EffectivePlatform)` there is only `$(MSBuildThisFileDirectory)$(EffectivePlatform)`.

**Version**
SDK: 1.0.818.41
Framework: MFC
OS: Win10

[AB#32801933](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/32801933)

**Comment by nikitavirg** [comment]
I have the same issue on my side.

**Comment by champnic** [comment]
Thanks for catching this @mariusbancila - to be honest I'm not sure why this issue hasn't come up on GitHub before, it feels like more folks should be running into this! I've opened a bug on our backlog to get this fixed. In the meantime, the workaround should be straightforward on your end, let me know if adding the missing native path to your project file doesn't work for you.

**Comment by mariusbancila** [comment]
I don't know why it hasn't come up on GitHub. When I downloaded the package, NuGet said 0 downloads so far. Anyways, with all due respect, this is becoming really annoying and I'm losing confidence in this project. We're waiting since October for a package that delivers a working build script. This version should have been the release that finally fixed the https://github.com/MicrosoftEdge/WebView2Feedback/issues/506 issue, but guess what, it comes with another error in the script. In the meantime, we need to rely on local manual changes or preview versions. Not cool.

**Comment by EikeAtOT** [comment]
@champnic : These things happen (though it should never have passed CI tests). What makes it a big issue though is that we have to wait for the next regular release in order to get such a minor fix. Is there really no way to deploy a hot fix?

**Comment by champnic** [comment]
This is fixed in both 1.0.902-prerelease and 1.0.864.35 SDK packages. Thanks!
